### ISSUE TITLE
Migrate CASA cabs to casatools 

### ISSUE THREAD

**Issue by SpheMakh** [caracal-pipeline/stimela#141]
The package is available on pypi!  as [casatools](https://pypi.org/project/casatools/)

I never thought I'd live to see the day

![Screenshot from 2023-03-27 18-28-51](https://user-images.githubusercontent.com/6624280/228005671-decccdde-8119-488e-bf3c-8e5110898827.png)

It should be an `IOError` but, baby steps.

**Comment by o-smirnov** [comment]
I don't think we need anything extra in Stimela to support this. The standard ``python`` flavour with e.g. ``command: (casatasks)listobs`` should work out of the box for this.

So this is really an issue that belongs on https://github.com/caracal-pipeline/cult-cargo, I'll transfer.
